content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst

```diff
@@ -0,0 +1,244 @@
+<https://sdk.preprod.invoicing.eta.gov.eg/codes/>`_ to code your business attributes.
+
+Most of these codes are handled automatically by Odoo, provided that your :ref:`branches
+<egypt/e-invoicing-branches>`, :ref:`customers <egypt/e-invoicing-customers>`, and :ref:`products
+<egypt/e-invoicing-products>` are correctly configured.
+
+- Company Information:
+
+  - Company Tax ID
+  - | Branch ID
+    | If you have only one branch, use ``0`` as the branch code.
+  - Activity type Code
+
+- Other Information:
+
+  - | Product Codes
+    | Your company's products should be coded and matched with their **GS1** or **EGS** codes.
+  - | Tax Codes
+    | Most of the taxes codes are already configured on Odoo in the :guilabel:`ETA Code (Egypt)`
+      field. We advise you to make sure these codes match your company's taxes.
+
+.. seealso::
+   - `Egyptian eInvoicing & eReceipt SDK - Code Tables
+     <https://sdk.preprod.invoicing.eta.gov.eg/codes/>`_
+   - :doc:`../../taxation/taxes/taxes`
+
+.. _egypt/e-invoicing-branches:
+
+Branches
+~~~~~~~~
+
+Create a contact and a journal for each branch of your company and configure its ETA settings.
+
+To do so, go to :menuselection:`Accounting --> Configuration --> Journals`, then click on
+:guilabel:`Create`.
+
+Name the journal according to your company's branch and set the :guilabel:`Type` as
+:guilabel:`Sales`. Next, open the :menuselection:`Advanced Settings` tab and fill out the
+:guilabel:`Egyptian ETA settings` section:
+
+- In the :guilabel:`Branch` field, select the branch's contact or create it.
+- Set the :guilabel:`ETA Activity Code`.
+- Set the :guilabel:`ETA Branch ID` (use ``0`` if you have one branch only).
+
+.. image:: egypt/branch-journal.png
+   :align: center
+   :alt: Sales journal configuration of an Egyptian company's branch
+
+.. important::
+   The contact selected in the :guilabel:`Branch` field must be set as a :guilabel:`Company`
+   (**not** as an :guilabel:`Individual`), and the :guilabel:`Address` and :guilabel:`Tax ID` fields
+   must be filled out.
+
+.. _egypt/e-invoicing-customers:
+
+Customers
+~~~~~~~~~
+
+Make sure your customers' contact forms are correctly filled out so your e-invoices are valid:
+
+- contact type: :guilabel:`Individual`: or :guilabel:`Company`:
+- :guilabel:`Country`:
+- :guilabel:`Tax ID`: Tax ID or Company registry for companies. National ID for individuals.
+
+.. note::
+   You can edit your customers' contact forms by going to :menuselection:`Accounting --> Customers
+   --> Customers`.
+
+.. _egypt/e-invoicing-products:
+
+Products
+~~~~~~~~
+
+Make sure your products are correctly configured so your e-invoices are valid:
+
+- :guilabel:`Product Type`: storable products, consumables, or services.
+- :guilabel:`Unit of Measure`: if you also use Odoo Inventory and have enabled :doc:`Units of
+  Measure <../../../../inventory_and_mrp/inventory/management/products/uom>`
```

Comment:
Since your relative path goes all the way back to the root of applications, you may want to do `/applications/...` instead.

Reply:
Ouais, je l'avais fait pour les chemins qui partent directement du root, mais okay :)